README.md

```diff
@@ -1,92 +1,60 @@
-- Autocomplete for common HubL tags, filters, and functions
-- Bracket wrapping, indentation, and other helpful editor features
+For fast HubSpot CMS Hub development right inside your editor.
 
-If you're new to CMS Hub, check out how to [get started with local development.](https://designers.hubspot.com/docs/tools/local-development)
+The new HubSpot panel in the activity bar provides developers access to features such as syntax formatters and highlighting for the [HubL templating language](https://designers.hubspot.com/docs/hubl/intro-to-hubl), as well as autocomplete for common HubL tags, filters, and functions. Additionally, it provides developers with quick views and commands to manage their connected accounts with HubSpot, extending some of the functionality of the HubSpot command line interface into the editor directly.
 
-## Getting Started
-This extension introduces new file languages: `HTML + HUBL` and `HTML + CSS`. In order to use these new features, you'll want to make sure you've told VSCode to use these languages. For one-off files, you can simply change the language in the lower right hand corner of your screen. However, we recommend adjusting your file associations under your User or Workspace preferences so HubL syntax highlighting will automatically be applied to all of your projects:
-- In VSCode, press `CMD` + `SHIFT` + `P` to open the command prompt
-- Search for and select the command `Preferences: Open Settings (UI)`
-- Choose either the "User" or "Workspace" tab
+<!-- TODO: Link 'HubSpot Developer Docs' text to CMS Dev Docs url when they are live -->
+To read more about this extension and its features, please see the documentation at the HubSpot Developer Docs. If you're new to CMS Hub, check out [how to get started with local development](https://designers.hubspot.com/docs/tools/local-development).
+
+## Settings
+
+### Language Modes
+
+This extension introduces new file languages: `HTML + HUBL` and `CSS + HUBL`. In order to use the full set of features from the extension, VS Code needs to be aware of these languages and how they are associated with various file types. For one-off files, you can simply change the Language Mode in the lower right hand status bar of the editor. However, we recommend adjusting your file associations under your User or Workspace preferences so HubL syntax highlighting will automatically be applied to all of your projects:
+
+- In VSCode, press `CMD + SHIFT + P` to open the command prompt
+- Search for and select the command `Preferences: Open User Settings`
+- Choose either the "User" or "Workspace" tab to apply these settings
 - In "Search settings" look up `files.associations`
 - Select "Add Item" and add these file associations: `*.html: html-hubl` and `*.css: css-hubl`
+- For more information about how VSCode settings work, [please read the official VS Code documentation](https://code.visualstudio.com/docs/getstarted/settings).
```

Comment:
We should automate this https://git.hubteam.com/HubSpot/cms-developer-tooling/issues/80